source/app/util/environment-check.ts

```diff
@@ -132,7 +132,14 @@ export default async function environmentCheck (): Promise<void> {
     }
   }
 
-  process.env.ZETTLR_IS_TRAY_SUPPORTED = (await isTraySupported()).value.toString()
+  try {
+    if (await isTraySupported()) {
+      process.env.ZETTLR_IS_TRAY_SUPPORTED = '1'
+    }
+  } catch (err) {
+    process.env.ZETTLR_IS_TRAY_SUPPORTED = '0'
+    global.log.warning(err)
```

Comment:
Just a note of caution:

* global.log.warning expects a simple string
* global.log.error expects a simple string as its first argument, but supports passing in an arbitrary object (in this case: The full error object) as its second argument.

So you need to change this to one of the following:

```js
global.log.warning(err.message) // If you don't need any more information

global.log.error(err.message, err) // In case we might need the stack trace and further info from the error object
```

Reply:
Thank you, good pick-up. I chose `global.log.warning`